boards/common/iotlab/include/board_common.h

```diff
@@ -49,46 +49,46 @@ extern "C" {
  * Tell the xtimer that we use a 16-bit peripheral timer
  * @{
  */
-#define XTIMER_WIDTH        (16U)
-#define XTIMER_OVERHEAD     (6U)
-#define XTIMER_SHOOT_EARLY  (3U)
+#define XTIMER_WIDTH              (16)
+#define XTIMER_OVERHEAD           (6)
+#define XTIMER_SHOOT_EARLY        (3)
 /** @} */
 
 /**
- * @brief   Define the interface to the AT86RF231 radio
+ * @name    Define the interface to the AT86RF231 radio
  *
  * {spi bus, spi speed, cs pin, int pin, reset pin, sleep pin}
+ * @{
  */
-#define AT86RF2XX_PARAMS_BOARD      {.spi       = SPI_DEV(0), \
-                                     .spi_clk   = SPI_CLK_5MHZ, \
-                                     .cs_pin    = GPIO_PIN(PORT_A, 4), \
-                                     .int_pin   = GPIO_PIN(PORT_C, 4), \
-                                     .sleep_pin = GPIO_PIN(PORT_A, 2), \
-                                     .reset_pin = GPIO_PIN(PORT_C, 1)}
+#define AT86RF2XX_PARAM_CS        GPIO_PIN(PORT_A, 4)
+#define AT86RF2XX_PARAM_INT       GPIO_PIN(PORT_C, 4)
+#define AT86RF2XX_PARAM_SLEEP     GPIO_PIN(PORT_A, 2)
+#define AT86RF2XX_PARAM_RESET     GPIO_PIN(PORT_C, 1)
+/** @} */
 
 /**
  * @name    LED pin definitions and handlers
  * @{
  */
-#define LED0_PIN            GPIO_PIN(PORT_D, 2)
-#define LED1_PIN            GPIO_PIN(PORT_B, 5)
-#define LED2_PIN            GPIO_PIN(PORT_C, 10)
+#define LED0_PIN                  GPIO_PIN(PORT_D, 2)
+#define LED1_PIN                  GPIO_PIN(PORT_B, 5)
+#define LED2_PIN                  GPIO_PIN(PORT_C, 10)
```

Comment:
here and in (probably) many other places of this PR: the indention is not aligned to 4-spaces anymore... It was before, but now the indention changed by 6 space chars